src/elements/Typography.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import { styled } from "../platform/primitives"
+import { color, maxWidth, space, textAlign, themeGet } from "styled-system"
+
+const dynamicTheme = callback => props =>
+  themeGet.apply(null, [].concat(callback(props)))(props)
+
+const selectFontType = ({ weight = "regular", italic }) => {
+  if (italic) {
+    if (weight === "medium") return "mediumItalic"
+    return "italic"
+  }
+  return weight
+}
+
+const fontPath = props => [
+  `fontFamily.${props.family}.${selectFontType(props)}`,
+  "regular", // fallback
+]
+
+const fontSize = props => `typeSizes.${props.typeSize}.fontSize`
+
+const lineHeight = props => `typeSizes.${props.typeSize}.lineHeight`
+
+const Text = styled.Text`
```

Comment:
@zephraph as @damassi mentioned, let's add an interface to that one as well. Something like,

```
interface TextProps {
  family: "unica" | "garamond" | "avantgarde"
  ...
}
```